src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -42,6 +42,43 @@ private[avro] object TestUtils {
    * copied from Spark with minor modifications made to it. See original source at:
    * github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala
    */
+
+  def checkReloadMatchesSaved(testFile: String, avroDir: String) = {
+
+    def convertToString(elem: Any): String = {
+      elem match {
+        case null => "NULL" // HashSets can't have null in them, so we use a string instead
+        case arrayBuf: ArrayBuffer[Any] => arrayBuf.toArray.deep.mkString(" ")
+        case arrayByte: Array[Byte] => arrayByte.deep.mkString(" ")
+        case other => other.toString
+      }
+    }
+
+    val originalEntries = TestSQLContext.avroFile(testFile).collect()
+    val newEntries = TestSQLContext.avroFile(avroDir).collect()
+
+    assert(originalEntries.size == newEntries.size)
+
+    val origEntrySet = new Array[HashSet[Any]](originalEntries(0).size)
+    for (i <- 0 until originalEntries(0).size) {origEntrySet(i) = new HashSet[Any]()}
```

Comment:
Whoops, I violated code style when this was written. Can you please change this to `{ origEntrySet(i) = new HashSet[Any]() }`. Thanks!